services/src/main/scala/cromwell/pipeline/service/InputNodeService.scala

```diff
@@ -0,0 +1,31 @@
+package cromwell.pipeline.service
+
+import cats.implicits._
+import cromwell.pipeline.datastorage.dao.repository.DocumentRepository
+import cromwell.pipeline.datastorage.dto.InputNode
+import org.mongodb.scala.Completed
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class InputNodeService(repository: DocumentRepository)(implicit ec: ExecutionContext) {
```

Comment:
Same thing about naming - may be better DocumentService?

Reply:
I can't name it like so, because this service contains specific method for working with nodes collection